plugin/adsk/scripts/mayaUsdTranslatorImport.mel

```diff
@@ -45,6 +45,7 @@ global proc int mayaUsdTranslatorImport (string $parent,
     int $bResult = 0;
     int $cw1 = 180;
     int $cw2 = 300;
+    int $bw = 125;
```

Comment:
Instead of just hiding the button, which continues to take up space, I'll now adjust the width to 1 just like we do with the text.  I also tried setting "manage" to false which is documented to not take up any space, but the rowLayout still has the column set to have a width and it doesn't adjust the way I'd like.

Reply:
And since we'll be explicitly flipping the width of the button we need to pick a regular width instead of going with the default, so adding a variable like some of the others.